generate-renode-scripts.py

```diff
@@ -362,28 +336,61 @@ def generate_repl():
         }
     }
 
-    shadow_base = (configuration.constants['shadow_base']['value']
-                   if 'shadow_base' in configuration.constants
-                   else None)
-
-    for mem_region in configuration.mem_regions.values():
-        if mem_region['name'] not in non_generated_mem_regions:
-            result += generate_memory_region(mem_region, shadow_base)
+    # RISC-V CPU in Renode requires memory region size
```

Comment:
I think this should probably be a FIXME / TODO?

Reply:
This is a known hard-limitation in Renode, not an effect of a bug. We have no plans to handle smaller memory regions right now.

Reply:
Sure. I extended the comment.